armi/resources/burn-chain.yaml

```diff
@@ -1,5 +1,4 @@
 AM241:
-- nuSF: 2.5
```

Comment:
Can you comment on why all of the nuSF values have been stripped out of this file?

Reply:
I guess it is to avoid duplication with the data now stored in the nuclides.dat file?

Reply:
Yes basically that data is in nuclides.dat now and this can be treated as a way to override data that is loaded in with this file. This goes for halflife as well.